desci-server/prisma/migrations/20240123184456_add_api_key/migration.sql

```diff
@@ -0,0 +1,22 @@
+-- CreateTable
+CREATE TABLE "ApiKey" (
+    "id" SERIAL NOT NULL,
+    "key" TEXT NOT NULL,
```

Comment:
Maybe call this `hashedKey`? Thought this was cleartext until I got to the create function.